tests/test_git_push.rs

```diff
@@ -179,6 +198,15 @@ fn test_git_push_unsnapshotted_change() {
     test_env.jj_cmd_success(&workspace_root, &["git", "push", "--change", "@"]);
     std::fs::write(workspace_root.join("file"), "modified").unwrap();
     test_env.jj_cmd_success(&workspace_root, &["git", "push", "--change", "@"]);
+    // test pushing two changes at once
+    std::fs::write(workspace_root.join("file"), "modified2").unwrap();
+    test_env.jj_cmd_success(&workspace_root, &["new"]);
+    test_env.jj_cmd_success(&workspace_root, &["describe", "-m", "bar"]);
```

Comment:
nit: you can combine these to `jj new -m bar`

Reply:
Good to know! I think I knew this once, but perhaps I should try actually using it.